.github/workflows/check.yml

```
@@ -78,6 +76,9 @@ jobs:
       - name: "ACTIONS: Checkout"
         uses: actions/checkout@v4
 
+      - name: "MOUNT: Logs path"
+        run: ln -s /mnt/root/fuzzer_logs $PWD/utils/runtime-fuzzer/fuzz/artifacts
```

`ln -s` accept args `existing_source_file optional_symbolic_link`
```suggestion
        run: |
          FUZZER_LOGS_PATH=/mnt/root/fuzzer_logs
          mkdir -p $FUZZER_LOGS_PATH
          ln -s $PWD/utils/runtime-fuzzer/fuzz/artifacts $FUZZER_LOGS_PATH/artifacts
          ln -s $PWD/utils/runtime-fuzzer/proptest-regressions $FUZZER_LOGS_PATH/proptest-regressions
```